.github/workflows/notebooks-release.yaml

```
@@ -0,0 +1,210 @@
+
+      - name: Check if PR is merged
+        id: check_pr
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # PR to look for
+          PR_TITLE="[Updater Action] Update BuildConfigs for CUDA RStudio and RStudio"
+
+          #Fetch matching PRs
+          PR_NUMBER=$(gh pr list --repo "$REPO_OWNER/$REPO_NAME" --state all --search "$PR_TITLE" --json number,title | jq -r '.[0].number')
+          echo "PR Numbers: $PR_NUMBER"
+
+          if [ -z "$PR_NUMBER" ]; then
+            echo "No PR found with title: $PR_TITLE"
+            exit 1
+          fi
+
+          # Polling loop to wait for the PR to be merged total timeout=5h
+          MAX_ATTEMPTS=30
+          SLEEP_DURATION=600
+
+          for (( i=1; i<=MAX_ATTEMPTS; i++ )); do
+            echo "Checking if PR #$PR_NUMBER is merged (Attempt $i/$MAX_ATTEMPTS)..."
+            PR_STATE=$(gh pr view --repo "$REPO_OWNER/$REPO_NAME" $PR_NUMBER --json mergedAt --jq '.mergedAt')
+
+            if [ "$PR_STATE" = "null" ] || [ -z "$PR_STATE" ]; then
+              echo "PR #$PR_NUMBER is not merged yet. Waiting..."
+              sleep $SLEEP_DURATION
+            else
+              echo "PR #$PR_NUMBER is merged!"
+              echo "pr_merged_b=true" >> $GITHUB_ENV
+              echo "pr_merged_b=true" >> $GITHUB_OUTPUT
+              exit 0
+            fi
+          done
+
+          echo "Timed out waiting for PR #$PR_NUMBER to be merged."
+          echo "pr_merged_b=false" >> $GITHUB_ENV
+          echo "pr_merged_b=false" >> $GITHUB_OUTPUT
+          exit 1
+
+  # 5. Generate the release with BuildConfigs if needed
+  Generate-release_with_buildconfigs:
+    needs: [Update-manifests, Manifests-merged, Update-buildConfigs, BuildConfigs-merged]
+    if: ${{ needs.Manifests-merged.outputs.pr_merged_m == 'true' && needs.BuildConfigs-merged.outputs.pr_merged_b == 'true' && github.event.inputs.update_buildconfigs == 'true' && github.event.inputs.release-type == 'Release' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Invoke Script to handle creating a release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        working-directory: ${{env.GITHUB_WORKSPACE}}
+        run: |
+          gh release create "$RELEASE_TAG" --title="$RELEASE_NAME-$RELEASE_TAG" --generate-notes --target $BRANCH
+
+  # 6. Generate the release without BuildConfigs
+  Generate-release:
+    needs: [Update-manifests, Manifests-merged]
+    if: ${{ needs.Manifests-merged.outputs.pr_merged_m == 'true' && github.event.inputs.update_buildconfigs == 'false' && github.event.inputs.release-type == 'Release' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Invoke Script to handle creating a release
```

```suggestion
      - name: Generate notebooks release
```

As there is no script that you invoke!